pkg/notes/notes.go

```diff
@@ -849,3 +879,52 @@ func prettifySIGList(sigs []string) string {
 
 	return sigList
 }
+
+// ApplyMap Modifies the content of the release using information from
+//  a ReleaseNotesMap
+func (rn *ReleaseNote) ApplyMap(noteMap *ReleaseNotesMap) error {
+
+	if noteMap.ReleaseNote.Author != nil {
+		rn.Author = *noteMap.ReleaseNote.Author
+		rn.AuthorURL = "https://github.com/" + *noteMap.ReleaseNote.Author
```

Comment:
We should make "https://github.com/" a const in the `github` package (can be a follow-up).